### PR TITLE
Update microsoft-365-vpn-implement-split-tunnel.md

### PR DESCRIPTION
**Pull request by PaulCollinge**
Updated FAQ to reflect support in browser

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/enterprise/microsoft-365-vpn-implement-split-tunnel.md [hunk 1]
```diff
@@ -323,7 +323,7 @@ Port 80 is only used for things like redirect to a port 443 session, no customer
 
 ### Does split-tunnel configuration work for Teams running in a browser?
 
-**No**, it does not. It works only on Microsoft Teams client version 1.3.00.13565 or greater. This version includes improvements in how the client detects available network paths.
+Yes it does, via supported browsers which are listed at https://docs.microsoft.com/en-us/microsoftteams/get-clients#web-client
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/microsoft-365-vpn-implement-split-tunnel.md:326]:
```suggestion
Yes, it does, via supported browsers which are listed in the [Web client](https://docs.microsoft.com/microsoftteams/get-clients#web-client) topic.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@PaulCollinge Please update with the feedback. Thanks

**Review by PaulCollinge — COMMENTED** [review]
Change looks good.

**Comment by JohanFreelancer9** [comment]
Hi @PaulCollinge - could you please commit my changes?

**Pull request merged**